test/utils/utils.ts

```diff
@@ -313,27 +313,28 @@ export async function depositV3(
   ]);
 
   const lastEvent = events.at(-1);
-  const args = lastEvent?.args;
+  let args = lastEvent?.args;
   assert.exists(args);
+  args = args!;
 
   const { blockNumber, transactionHash, transactionIndex, logIndex } = lastEvent!;
 
   return {
-    depositId: toBN(args!.depositId),
+    depositId: toBN(args.depositId),
     originChainId: Number(originChainId),
-    destinationChainId: Number(args!.destinationChainId),
-    depositor: args!.depositor,
-    recipient: args!.recipient,
-    inputToken: args!.inputToken,
-    inputAmount: args!.inputAmount,
-    outputToken: args!.outputToken,
-    outputAmount: args!.outputAmount,
-    quoteTimestamp: args!.quoteTimestamp,
-    message: args!.message,
-    messageHash: getMessageHash(args!.message),
-    fillDeadline: args!.fillDeadline,
-    exclusivityDeadline: args!.exclusivityDeadline,
-    exclusiveRelayer: args!.exclusiveRelayer,
+    destinationChainId: Number(args.destinationChainId),
+    depositor: args.depositor,
+    recipient: args.recipient,
+    inputToken: args.inputToken,
+    inputAmount: args.inputAmount,
+    outputToken: args.outputToken,
+    outputAmount: args.outputAmount,
+    quoteTimestamp: args.quoteTimestamp,
+    message: args.message,
+    messageHash: getMessageHash(args.message),
+    fillDeadline: args.fillDeadline,
+    exclusivityDeadline: args.exclusivityDeadline,
+    exclusiveRelayer: args.exclusiveRelayer,
```

Comment:
This is just a driveby improvement that I spotted when fixing the neighbouring code for this change.